packages/suricata.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,114 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+Import-Module powershell-yaml
+
+# set configurations
+$toolName = "Suricata"
```

Comment:
According to [our documentation](https://github.com/mandiant/VM-Packages/wiki/Coding-Conventions#toolname), the tool name is the file name with the `.exe` which in this case seems to be `suricata.exe`
```suggestion
$toolName = "suricata"
```